client/src/components/job-application-component/application-details-card/index.jsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react';
+import PropTypes from "prop-types";
+import {
+  Card,
+  CardContent,
+  Typography,
+} from "@mui/material";
+import { Divider } from "../index.styles";
+
+import BoxRowComponent from "../box-row-component";
+
+function ApplicationDetailsCard({ application, jobPost, setApplicationStatus }) {
+
+  const jobTitle = useState(jobPost.title);
+  const employerName = useState(jobPost.employer);
+  const dateApplied = useState(application.createdAt);
```

Comment:
please use the client/src/utils/date.js function to format the date applied object!